src/loader/loader.go

```diff
@@ -51,6 +52,11 @@ func Load(opts *LoaderOptions) (*types.Project, error) {
 	// If DryRun is set to validate the config, then force IsStrict to true:
 	mergedProject.IsStrict = opts.DryRun || mergedProject.IsStrict
 
+	// Override log level if given in env
+	if envLevel, set := os.LookupEnv(config.LogLevelEnvVarName); set {
+		mergedProject.LogLevel = envLevel
```

Comment:
The level should be validated, please do it at the validators.go location that I sent you.

Reply:
My bad.
You are right.
Note to self: don't check PRs while doing 7 other things 😄